### DISCUSSION TITLE
where should I put DeleteAction ?

### DISCUSSION THREAD

**Discussion by magic-thomas** [filamentphp/filament#8745] [Help] [answered]
Hi. 
I still using v2. 

I would like put my code after DeleteAction, in Editpage.

I see this in 'https://filamentphp.com/docs/2.x/admin/resources/deleting-records'

```php 
DeleteAction::make()
    ->before(function () {
        // ...
    })
    ->after(function () {
        // ...
    })

```` 

But where should I put this code? in which page, in which function ?

**Comment by VincvanAs** [comment, marked as answer]
protected function getActions(): array
 {
        return [
            Actions\DeleteAction::make(),
        ];
}

This function you mean?

**Reply by magic-thomas** [reply]
Thanks for help. 

I wonder why there is no comments about that in documents.
